packages/pwashortcut.rb

```diff
@@ -0,0 +1,169 @@
+require 'package'
+
+class Pwashortcut < Package
+  description 'Scripts to create a PWA shortcut on ChromeOS dock for Chromebrew GUI Apps.'
+  homepage 'https://github.com/supechicken666/pwashortcut/'
+  version '1.0'
+  source_url 'https://github.com/supechicken666/pwashortcut/raw/master/brew_transparent.png'
+  source_sha256 'dedaa7b9dda61413ff03c823f035bebf5a006dc6a4c37eb8bbd27363e676fd56'
+
+  def self.build
+    system "pip install flask"
+    system "cat <<'EOF'> main.py.bak
+from flask import Flask, render_template, redirect, url_for, send_from_directory
+import os
+
+app = Flask(__name__)
+
+EOF"
+    #######################################
+    system "cat <<'EOF'> installer.html.bak
+<!DOCTYPE html>
+<html>
+<head>
+<title>Shortcut Installer</title>
+<link rel=\"manifest\" href=\"manifest.json\" />
+</head>
+<body style=\"background-color:powderblue;\">
+    <h1>Shortcut Installer</h1>
+    <p>Add Linux GUI App shortcut to your dock</p>
+    <h2>HowTo</h2>
+    <h3>Step 1</h3>
+    <p>Click the ••• icon in the right corner of the screen</p>
+    <h3>Step 2</h3>
+    <p>Select 'Create shortcut...' under 'More tools'</p>
+    <h3>Step 3</h3>
+    <p>Tick the 'Open as window' checkbox</p>
+    <h3>Step 4</h3>
+    <p>Simply click the blue 'Create' button!</p>
+    <h3>Step 5</h3>
+    <p>Close this window and reopen it by the new shortcut</p>
+    <h2>That's all:) </h2>
+</body>
+</html>
+EOF"
+    #######################################
+    system "cat <<'EOF'> manifest.json.bak
+v1
+  \"name\"v4 \"linuxapp\"v6
+  \"short_name\"v4 \"linuxapp\"v6
+  \"display\"v4 \"minimal-ui\"v6
+  \"start_url\"v4 \"unixapp.app\"v6
+  \"theme_color\"v4 \"#000000\"v6
+  \"background_color\"v4 \"#000000\"v6
+  \"icons\"v4 v2
+    v1
+      \"src\"v4 \"brew.png\"v6
+      \"sizes\"v4 \"200x200\"v6
+      \"type\"v4 \"image/png\"
+    v9
+  v8
+v9
+EOF"
+    system "sed -i 's/v6/,/g' manifest.json.bak"
+    system "sed -i 's/v4/:/g' manifest.json.bak"
+    system "sed -i 's/v1/\{/g' manifest.json.bak"
+    system "sed -i 's/v2/\[/g' manifest.json.bak"
+    system "sed -i 's/v9/\}/g' manifest.json.bak"
+    system "sed -i 's/v8/]/g' manifest.json.bak"
+    #######################################
+    system "cat <<'EOF'> pwashortcut
+#!/bin/bash
+# A simple start script
+export CREW_PREFIX=#{CREW_PREFIX}
+export PYTHONPATH=$CREW_PREFIX/lib/python3.9/site-packages/
+export tools=$CREW_PREFIX/lib/pwa/tools
+export PREFIX=$CREW_PREFIX/lib/pwa
```

Comment:
Might be good to create `PWA_PREFIX` instead to avoid a potential environment variable conflict.